websites/H/HiAnime/presence.ts

```diff
@@ -147,7 +152,13 @@ presence.on("UpdateData", async () => {
 			[presenceData.startTimestamp, presenceData.endTimestamp] =
 				presence.getTimestamps(data.currTime, data.duration);
 			presenceData.smallImageKey = Assets.Play;
-		} else if (data) presenceData.smallImageKey = Assets.Pause;
+			presenceData.smallImageText = (await strings).play;
+		} else if (data) {
+			presenceData.smallImageKey = Assets.Pause;
+			presenceData.smallImageText = (await strings).pause;
+			delete presenceData.startTimestamp;
+			delete presenceData.endTimestamp;
+		}
 
```

Comment:
Put in this if statement 
```suggestion
				presenceData.type = ActivityType.Watching
```